instrumentation/rack/lib/opentelemetry/instrumentation/rack/middlewares/tracer_middleware.rb

```diff
@@ -76,6 +83,10 @@ def call(env) # rubocop:disable Metrics/AbcSize
 
           private
 
+          def untraced_endpoint?(path)
```

Comment:
Summarizing a bit of our discussion here from the SIG meting here:

I expressed interest in some flexibility here that would allow users to deal with more complex `PATH_INFO`, e.g.
 
- Allowing for regular expressions to so that users can match more complex resource oriented paths
- A lambda/proc with custom logic

I believe @robertlaurin brought up that in addition to `PATH_INFO` we may in the future want to inspect other parts of the request, e.g. an HTTP Header.

I think the consensus was that starting with supporting string paths would be fine for covering the use case of disabling tracing for static health endpoints. 

Does that sound accurate to y'all @robertlaurin @fbogsany @mwear?